src/common/TrackedOp.cc

```diff
@@ -175,6 +175,12 @@ void OpTracker::unregister_inflight_op(TrackedOp *i)
 
 bool OpTracker::check_ops_in_flight(std::vector<string> &warning_vector)
 {
+  int slow;
```

Comment:
Initialize this (unlike the case below this one doesn't matter so much, but still it's cleaner)